frank-flow/src/frontend/src/assets/styles/ngx-smart-modal.scss

```diff
@@ -34,6 +34,52 @@
         display: flex;
         justify-content: end;
         gap: 16px;
+        background-color: var(--bg-color);
+      }
+
+      button {
+        background-color: var(--bg-color);
+      }
```

Comment:
There is a button component. The style for the whole application should match, or we should consider other options if it doesn't look good.